packages/core/src/server/middlewares/resource.ts

```diff
@@ -118,8 +118,8 @@ export function findResource(
     publicPath,
     req.url
   );
-
-  const resource = compiler.resource(resourceWithoutPublicPath);
+  const normalizedPath = resourceWithoutPublicPath.replace(/^\//, '');
```

Comment:
```suggestion
  const normalizedPath = resourceWithoutPublicPath.startsWith('/') ? resourceWithoutPublicPath.slice(1) : resourceWithoutPublicPath;
```

Regular expression may cost more unexpected overheads, I think we should avoid using regex. WDYT?

Reply:
Performance may not be a top priority in the server, but you are right. Here I've thought of another method. Let me give it a try.